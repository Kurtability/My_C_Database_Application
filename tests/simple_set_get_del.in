del a
get a
set a 1 2 3
get a
del a
get a
bye
