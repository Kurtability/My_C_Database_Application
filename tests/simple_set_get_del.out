> del a 
no such key

> get a 
no such key

> set a 1 2 3 
ok

> get a 
[1 2 3]

> del a
ok

> get a 
no such key

> bye
bye
